### PR TITLE
Allow APM32 MCUs on certain CannonKeys PCBs

### PR DESCRIPTION
**Pull request by awkannan**
## Description

Allow APM32 MCUs on certain CannonKeys PCBs
We'll do this by removing the DFU suffix which locks in the firmware to just STM32 device ids.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Didn't we want `FFFF` for the VID/PID combo, as this seems to work with dfu-util?

**Comment by awkannan** [comment]
Yup, I haven't had a chance to update my PRs since that convo - I will make the changes

File: keyboards/cannonkeys/an_c/rules.mk [hunk 1]
```diff
@@ -20,6 +20,7 @@ WS2812_DRIVER = spi
 
 LAYOUTS = 60_ansi 60_tsangan_hhkb
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/an_c/rules.mk:23]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/db60/rules.mk [hunk 2]
```diff
@@ -17,5 +17,7 @@ WS2812_DRIVER = spi
 
 LAYOUTS = 60_ansi 60_tsangan_hhkb 60_iso
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/db60/rules.mk:20]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/devastatingtkl/rules.mk [hunk 3]
```diff
@@ -16,5 +16,7 @@ BACKLIGHT_ENABLE = yes
 RGBLIGHT_ENABLE = yes
 WS2812_DRIVER = spi
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/devastatingtkl/rules.mk:19]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/instant60/rules.mk [hunk 4]
```diff
@@ -19,6 +19,7 @@ WS2812_DRIVER = spi
 
 LAYOUTS = 60_ansi 60_tsangan_hhkb
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/instant60/rules.mk:22]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/instant65/rules.mk [hunk 5]
```diff
@@ -21,3 +21,5 @@ WS2812_DRIVER = spi
 
 # Enter lower-power sleep mode when on the ChibiOS idle thread
 OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
+
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/instant65/rules.mk:25]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/obliterated75/rules.mk [hunk 6]
```diff
@@ -19,5 +19,7 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 WS2812_DRIVER = spi
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/obliterated75/rules.mk:22]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/rekt1800/rules.mk [hunk 7]
```diff
@@ -15,5 +15,7 @@ CUSTOM_MATRIX = no          # Custom matrix file
 BACKLIGHT_ENABLE = yes
 RGBLIGHT_ENABLE = no
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/rekt1800/rules.mk:18]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/satisfaction75/rules.mk [hunk 8]
```diff
@@ -24,6 +24,7 @@ QWIIC_ENABLE += MICRO_OLED
 
 DEFAULT_FOLDER = cannonkeys/satisfaction75/rev1
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/satisfaction75/rules.mk:27]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/savage65/rules.mk [hunk 9]
```diff
@@ -21,4 +21,6 @@ WS2812_DRIVER = spi
 # Enter lower-power sleep mode when on the ChibiOS idle thread
 OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
 
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/savage65/rules.mk:24]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

File: keyboards/cannonkeys/tmov2/rules.mk [hunk 10]
```diff
@@ -20,3 +20,5 @@ WS2812_DRIVER = spi
 
 # Enter lower-power sleep mode when on the ChibiOS idle thread
 OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
+
+DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```
Comment (zvecr) [inline keyboards/cannonkeys/tmov2/rules.mk:24]:
```suggestion
# Wildcard to allow APM32 MCU 
DFU_SUFFIX_ARGS = -p FFFF -v FFFF
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**